tools/README.md

```diff
@@ -128,7 +129,17 @@ echo '9JsCkzFH/W7SbgvgokeJTsckJs/uoAPWWwC40Y6UfFo=' | dscp-keygen --secret --com
 echo '9JsCkzFH/W7SbgvgokeJTsckJs/uoAPWWwC40Y6UfFo=' | dscp-keygen --secret --command "educator-auth:/api/student/v1/courses"
 ```
 You can later append token with
-* `-H "Authorization: Bearer <token>"` for `curl`
-* `Authorization:Bearer\ <token>` to [`http`](https://httpie.org/).
+1. `-H "Authorization: Bearer <token>"` for `curl`
+2. `Authorization:Bearer\ <token>` for [`http`](https://httpie.org/).
+
+* Submit a new submission from student - owner of given secret key:
+
+```base
+echo 456 \
+    | dscp-keygen --seed --command student-submission:3656234 \
```

Comment:
Person reading the README will not get what is this number. Point out that this is the random seed.